Code/Tools/AssetProcessor/native/ui/BuilderInfoPatternsModel.cpp

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
```

Comment:
no use for **"#pragma once"** in cpp files

Reply:
Feeling ashamed for letting this happen again. I will fix that and check thoroughly.

Reply:
resolved